python-client/build.gradle

```diff
@@ -0,0 +1,7 @@
+plugins {
```

Comment:
what are these `python-client/` files related to/used for?  Is this for like local development only?

Reply:
yep, for local development/testing only. documentation is in the other repo, but the update to README here points to that

Reply:
want me to add a README in here to clarify that?